src/lib/expansion/expansion-panel.scss

```diff
@@ -0,0 +1,31 @@
+@import '../core/style/variables';
+@import '../core/style/elevation';
+
+.mat-expansion-panel {
+  @include mat-elevation-transition;
+  box-sizing: content-box;
+  display: block;
+
+  &:not([class*='mat-elevation-z']) {
+    @include mat-elevation(2);
+  }
+}
+
+.mat-expansion-panel-body {
+  overflow: hidden;
+  padding: 0 24px;
+}
+
+[mat-action-row],
+[md-action-row] {
+  border-top: 1px solid rgba(0, 0, 0, 0.12);
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  margin: 10px -24px 0 -24px;
```

Comment:
It was originally needed because a weird spacing issue, it essentially, just offset the containers padding.  I was able to adjust it to no longer be needed.